blocks/api/test/parser.js

```diff
@@ -97,11 +98,11 @@ describe( 'block parser', () => {
 		it( 'returns false is block is not valid', () => {
 			registerBlockType( 'core/test-block', defaultBlockSettings );
 
-			expect( isValidBlock(
+			expect( () => isValidBlock(
 				'Apples',
 				getBlockType( 'core/test-block' ),
 				{ fruit: 'Bananas' }
-			) ).toBe( false );
+			) ).toThrow();
```

Comment:
Should we test a specific error message here, or save this for later when we're ready to expose this info to the UI?